addon/components/bs-form.js

```diff
@@ -400,7 +409,7 @@ export default class Form extends Component {
 
     return RSVP.resolve()
       .then(() => {
-        return this.hasValidator ? this.validate(model) : null;
+        return this.hasValidator ? this.validate(model, this._element) : null;
```

Comment:
Somehow my review comment got lost. :cry:

I'm wondering if we could pick up the `form` element from the `event` instead of tracking it through `{{ref}}`. The event listener is registered on the `form` element. Shouldn't the `form` element be available as `event.target` in that case?

Reply:
Yes, indeed I had it this way, until I realized some existing test failed. It was one where we submit the form using the yielded submit action, and *not* a submit (browser) event. In which case we don't have `event.target`. That's why I also added the second test in this PR, which is replicating this very situation!

Reply:
Good catch. :clap: